gcloud-java-dns/src/main/java/com/google/gcloud/dns/ChangeRequestInfo.java

```diff
@@ -0,0 +1,358 @@
+      return this;
+    }
+
+    @Override
+    public Builder removeDeletion(RecordSet recordSet) {
+      this.deletions.remove(recordSet);
+      return this;
+    }
+
+    @Override
+    public ChangeRequestInfo build() {
+      return new ChangeRequestInfo(this);
+    }
+
+    @Override
+    Builder id(String id) {
+      this.id = checkNotNull(id);
+      return this;
+    }
+
+    @Override
+    Builder startTimeMillis(long startTimeMillis) {
+      this.startTimeMillis = startTimeMillis;
+      return this;
+    }
+
+    @Override
+    Builder status(ChangeRequestInfo.Status status) {
+      this.status = checkNotNull(status);
+      return this;
+    }
+  }
+
+  ChangeRequestInfo(BuilderImpl builder) {
+    this.additions = ImmutableList.copyOf(builder.additions);
+    this.deletions = ImmutableList.copyOf(builder.deletions);
+    this.id = builder.id;
+    this.startTimeMillis = builder.startTimeMillis;
+    this.status = builder.status;
+  }
+
+  /**
+   * Returns an empty builder for the {@code ChangeRequestInfo} class.
+   */
+  public static Builder builder() {
+    return new BuilderImpl();
+  }
+
+  /**
+   * Creates a builder populated with values of this {@code ChangeRequestInfo}.
+   */
+  public Builder toBuilder() {
+    return new BuilderImpl(this);
+  }
+
+  /**
+   * Returns the list of {@link RecordSet}s to be added to the zone upon submitting this change
+   * request.
+   */
+  public List<RecordSet> additions() {
+    return additions;
+  }
+
+  /**
+   * Returns the list of {@link RecordSet}s to be deleted from the zone upon submitting this change
+   * request.
+   */
+  public List<RecordSet> deletions() {
+    return deletions;
+  }
+
+  /**
+   * Returns the id assigned to this {@code ChangeRequest} by the server.
+   */
+  public String id() {
+    return id;
+  }
+
+  /**
+   * Returns the time when this {@code ChangeRequest} was started by the server.
+   */
+  public Long startTimeMillis() {
+    return startTimeMillis;
+  }
+
+  /**
+   * Returns the status of this {@code ChangeRequest}.
+   */
+  public ChangeRequestInfo.Status status() {
+    return status;
+  }
+
+  com.google.api.services.dns.model.Change toPb() {
+    com.google.api.services.dns.model.Change pb =
+        new com.google.api.services.dns.model.Change();
+    // set id
+    if (id() != null) {
+      pb.setId(id());
+    }
+    // set timestamp
+    if (startTimeMillis() != null) {
+      pb.setStartTime(ISODateTimeFormat.dateTime().withZoneUTC().print(startTimeMillis()));
+    }
+    // set status
+    if (status() != null) {
+      pb.setStatus(status().name().toLowerCase());
+    }
+    // set a list of additions
+    pb.setAdditions(Lists.transform(additions(), RecordSet.TO_PB_FUNCTION));
+    // set a list of deletions
+    pb.setDeletions(Lists.transform(deletions(), RecordSet.TO_PB_FUNCTION));
+    return pb;
+  }
+
+  static ChangeRequestInfo fromPb(com.google.api.services.dns.model.Change pb) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'com.google.api.services.dns.model.Change' due to existing import 'com.google.api.services.dns.model.Change'](https://www.codacy.com/app/mziccard/gcloud-java/file/2311199855/issues/source?bid=3123106&fileBranchId=3198278#l316)